xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -934,8 +931,8 @@ void CPVRClients::ProcessMenuHooks(int iClientID, PVR_MENUHOOK_CAT cat, const CF
       if (selection >= 0)
       {
         itrClients = clients.begin();
-        for (int i = 0; i < selection; i++)
-          itrClients++;
+        for (int i = 0; i < selection; ++i)
+          ++itrClients;
         iClientID = itrClients->first;
```

Comment:
Imo this highly depends on how often one uses <code>at</code> compared to iterators. Here, the ratio is "healthy" and in this case there is no relevant perfomance impact, if you ask me.